README.md

```diff
@@ -4,15 +4,47 @@ These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are r
 
 ## For developers
 
-OpenJDK Source Control repositories are managed using Mercurial. The scripts within this project mirror those Mercurial repositories
-to github.com/adoptium/jdk\<NN\>u , which then serves as a mirror for Adoptium to build the Hotspot variants, as well as providing branches for adding any Adoptium vendor "patches" needed.
+OpenJDK Source Control repositories are now managed using the GitHub (project Skara). The scripts within the projects that use Mercurial repositories
```

Comment:
I think only the first sentence needs to be kept

Reply:
Alright then

Reply:
Hello @karianna I have resolved it.